configure

```diff
@@ -76,6 +76,7 @@ OPTIONS=(
   "cclang_threadsan:no"
   "gperftools:no"
   "execinfo:auto"
+  "year2038:no"
```

Comment:
I'm not sure how exactly the build-system works, but afaik this allows you to set a default and manual override? Is this actually needed at all?